docs/content/en/docs/intro-klt.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introduction to the Keptn Lifecycle Toolkit
+linktitle: Introduction to the Keptn Lifecycle Toolkit
+description: Understand the Keptn Lifecycle Toolkit
+weight: 05
+cascade:
+  github_subdir: "docs/content/en/docs"
+  path_base_for_github_subdir: "/content/en/docs-dev"
+---
+
+The Keptn Lifecycle Toolkit (KLT)  makes your deployments observable,
+brings application-awareness to your Kubernetes cluster,
+and helps you reliably deliver your application with:
+
+* Pre-Deployment Tasks: e.g. checking for dependant services,
+  checking if the cluster is ready for the deployment, etc.
+* Pre-Deployment Evaluations: e.g. evaluate metrics
+  before your application gets deployed (e.g. layout of the cluster)
+* Post-Deployment Tasks: e.g. trigger a test,
+  trigger a deployment to another cluster, etc.
+* Post-Deployment Evaluations: e.g. evaluate the deployment,
+  evaluate the test results, etc.
+
+All of these things can be executed on a workload or on an application level,
+whereby a Keptn application is a collection of multiple workloads.
+
+## Compare KLT and Keptn V1-LTS
+
+The Keptn Lifecycle Controller (KLT) is an incubating CNCF project
```

Comment:
Keptn is an incubating CNCF project and the LT is a subproject of Keptn

Reply:
I agree with @thschue. I think a better wording would be
```suggestion
The Keptn Lifecycle Controller (KLT) is a Keptn subproject
```

Reply:
done and I learned something.  And I must get that "word list" started!